src/Requests.jl

```diff
@@ -153,6 +156,11 @@
         r.finished = true
         close(p.sock)
 
+        #If MIME type allows, convert r.data to a Julia string
+        mimetype = r.headers["Content-Type"]
+        if (contains(mimetype, "charset=utf-8") || contains(mimetype, "charset=us-ascii"))
```

Comment:
Although reading the specs more carefully say:
- charset specifications [must be case-insensitive](http://tools.ietf.org/html/rfc2046#section-4.1.2),
- the default `charset` for `text/` types registered prior to [rfc 6657](http://tools.ietf.org/html/rfc6657#section-4) is `us-ascii` and those after rfc 6657 have no default charset,
- `text/*` mimetypes that can specify an encoding internally are assumed to be in the encoding specified in the response (but this seems like too much too handle in Requests; I'd rather just give a byte buffer if it's ambiguous).